dictionaries/pt_br.dictionary.itop.ui.php

```diff
@@ -66,10 +66,10 @@
 	'Class:Query/Attribute:name+' => 'Identificar a consulta',
 	'Class:Query/Attribute:description' => 'Descrição',
 	'Class:Query/Attribute:description+' => 'Descrição longa para a consulta (finalidade, uso, etc)',
-	'Class:Query/Attribute:is_template' => 'Template for OQL fields~~',
-	'Class:Query/Attribute:is_template+' => 'Usable as source for recipient OQL in Notifications~~',
-	'Class:Query/Attribute:is_template/Value:yes' => 'Yes~~',
-	'Class:Query/Attribute:is_template/Value:no' => 'No~~',
+	'Class:Query/Attribute:is_template' => 'Template para campos OQL~~',
+	'Class:Query/Attribute:is_template+' => 'Utilizável como origem para o Destinatário OQL em Notificações~~',
+	'Class:Query/Attribute:is_template/Value:yes' => 'Sim~~',
+	'Class:Query/Attribute:is_template/Value:no' => 'Não~~',
```

Comment:
```suggestion
	'Class:Query/Attribute:is_template' => 'Template para campos OQL',
	'Class:Query/Attribute:is_template+' => 'Utilizável como origem para o Destinatário OQL em Notificações',
	'Class:Query/Attribute:is_template/Value:yes' => 'Sim',
	'Class:Query/Attribute:is_template/Value:no' => 'Não',
```

Please remove `~~` at the end of translated entries. The `~~` are here to help translators find the untranslated entries :)
Same comment for all other entries below, I didn't put a comment on each of them.